assets/app/scripts/controllers/newfromtemplate.js

```diff
@@ -0,0 +1,66 @@
+'use strict';
+
+/**
+ * @ngdoc function
+ * @name openshiftConsole.controller:NewFromTemplateController
+ * @description
+ * # NewFromTemplateController
+ * Controller of the openshiftConsole
+ */
+angular.module('openshiftConsole')
+  .controller('NewFromTemplateController', function ($scope, $http, $filter, LabelFilter) {
+
+    function imageItems(data) {
+      var images = []
+      data.items.forEach(function(item) {
+        if (item.kind == "ImageRepository") {
+          images.push(item)
+        }
+      })
+      return images
+    }
+    
+    $scope.createFromTemplate = function() {
+      //TODO: Submit template data
+    }
+    
+    $scope.cancel = function() {
+      $window.history.back();
```

Comment:
we'll probably need this in other places, might be nice to just make this an attribute directive http://stackoverflow.com/a/23351154